fides/templates/fides/fides-deployment.yaml

```diff
@@ -67,7 +67,7 @@ spec:
           - name: {{ $volume }}
             mountPath: {{ $configPath }}
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.fides.resources | nindent 12 }}
```

Comment:
Can you / should you make this backwards compatible?

Reply:
I tried making it `{{- toYaml (coalesce .Values.fides.resources .Values.resources) | nindent 12 }}` and the intellisense complained, but then I ran `helm lint` and it succeeded, so I wasn't sure.

Reply:
To be clear though, this will not be backwards compatible. It will require a change in how anyone calls this version of the Helm chart and above if they are specifying resources.